dcat/ngsild-payloads/predictionDataServiceProcessing.jsonld

```diff
@@ -9,16 +9,26 @@
         "type": "Relationship",
         "object": "urn:ngsi-ld:Theme:TrafficFlowPrediction"
     },
-    "refDataService": {
+    "instanceOf": {
         "type": "Relationship",
         "object": "urn:ngsi-ld:DataService:TrafficFlowPrediction:XYZ"
     },
+    "additionalConfig": {
+        "type": "JsonProperty",
+        "json": {
+            "startTime": "2023-02-01T00:00:00Z",
+            "endTime": "2023-02-08T00:00:00Z"
+        }
+    },
     "subscriptionData": {
-        "entities": [
-            {
-                "type": "TrafficFlowObserved"
-            }
-        ],
-        "q": "refCity==urn:ngsi-ld:City:IssyLesMoulineaux"
+        "type": "JsonProperty",
+        "json": {
+            "entities": [
+                {
+                    "type": "TrafficFlowObserved"
+                }
+            ],
+            "q": "refCity==urn:ngsi-ld:City:IssyLesMoulineaux"    
```

Comment:
What is this "q" attribute for ?

Reply:
it is for further enhancements in the data processing registration (see "Register a processing (5)" in the doc)